src/FsAutoComplete.Core/Commands.fs

```diff
@@ -74,19 +75,21 @@ type Commands (serialize : Serializer) =
                 [ Response.errors serialize (errors, file)]
     }
 
-    member __.FileChecked =
-        checker.FileChecked
-        |> Event.map (fun fn ->
-            let file = Path.GetFullPath fn
-            let res = state.FileCheckOptions |> Seq.tryFind (fun kv -> Path.GetFullPath kv.Key = file)
-            match res with
-            | None  -> async { return [Response.info serialize ( sprintf "Project for file not found: %s" file) ]  }
-            | Some kv ->
-                async {
-                    let! (_, checkResults) = checker.GetBackgroundCheckResultsForFileInProject(fn, kv.Value)
-                    return [ Response.errors serialize (checkResults.Errors, file) ] })
-
-
+    // member __.FileChecked =
+    //     checker.FileChecked
+    //     |> Event.map (fun fn ->
+    //         let file = Path.GetFullPath fn
+    //         let res = state.FileCheckOptions |> Seq.tryFind (fun kv -> Path.GetFullPath kv.Key = file)
+    //         match res with
+    //         | None  -> async { return [Response.info serialize ( sprintf "Project for file not found: %s" file) ]  }
+    //         | Some kv ->
+    //             async {
+    //                 let result= checker.TryGetRecentCheckResultsForFile(fn, kv.Value)
+    //                 return
+    //                     match result with
+    //                     | None -> [Response.info serialize "File not parsed"]
+    //                     | Some res -> [ Response.errors serialize (res.GetCheckResults.Errors, file) ]
+    //             })
```

Comment:
It was part of push notifications system I tried to use, which was unfortunately not working in the end. I hope to get back to it in the future.